drivers/scd30/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_SCD30
```

Comment:
Since you already did this kind of cleanup for `si1133` do you mind removing the unneeded `ifneq (,$(filter scd30,$(USEMODULE)))` in the `Makefile.dep`? You can obviously dismiss

Reply:
Sure!